src/Features/Core/Portable/ExtractMethod/MethodExtractor.cs

```diff
@@ -29,6 +29,8 @@ internal abstract partial class MethodExtractor(
         ExtractMethodGenerationOptions options,
         bool localFunction)
     {
+        public static readonly SyntaxAnnotation InsertionPointAnnotation = new();
```

Comment:
>InsertionPointAnnotation [](http://example.com/codeflow?start=48&length=24)

Might be worth making a PR to remove the nullable pragmas if this code is all getting cleaned up anyways.

Reply:
sure.  can do in followup.